gulp-tasks/reference-docs/filter-tags-to-build.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const fs = require('fs-extra');
+const path = require('path');
+
+const filterTagsToBuild = (tags, docPath) => {
```

Comment:
Please document functions with JSDoc. Include at least a brief description, and `@param`/`@return` as appropriate.  Will still merge if not added.